src/temp/ProjectListControls/ActiveSearchField.tsx

```diff
@@ -0,0 +1,63 @@
+import { useState, ChangeEvent, useEffect } from 'react';
+import { SearchTextField } from './SearchTextField';
+import CrossIcon from '../icons/CrossIcon';
+import style from './ProjectListControls.module.scss';
+import SearchIcon from '../icons/SearchIcon';
+import { useTranslations } from 'next-intl';
+import { SetState } from '../../features/common/types/common';
+import { useDebouncedEffect } from '../../utils/useDebouncedEffect';
+
+interface ActiveSearchFieldProps {
+  setIsSearching: SetState<boolean>;
+  setIsFilterOpen: SetState<boolean>;
+}
+
+const ActiveSearchField = ({
+  setIsSearching,
+  setIsFilterOpen,
+}: ActiveSearchFieldProps) => {
+  const [searchValue, setSearchValue] = useState('');
+  const [debouncedSearchValue, setDebouncedSearchValue] = useState('');
+
+  useEffect(() => {
+    if (debouncedSearchValue) window.alert(`${debouncedSearchValue} searched!`);
+  }, [debouncedSearchValue]);
+
+  useDebouncedEffect(
+    () => {
+      setDebouncedSearchValue(searchValue);
+    },
+    1000,
+    [searchValue]
+  );
+  const t = useTranslations('AllProjects');
+
+  const resetSearchTab = () => {
+    setSearchValue('');
+    setIsSearching(false);
+    setIsFilterOpen(false);
+  };
+
+  return (
+    <div className={style.projectListControls}>
```

Comment:
Should the `projectListControls` class be used here?

Reply:
Yes, Since I am using the same component (`ActiveSearchField`) for both mobile and large screens, the `projectListControls` class should handle the styling for both.

Reply:
`ProjectListControls` is the name of the main component, so using the `projectListControls` class name for the container here will cause confusion and interdependability of styles. I suggest using a different class name for the `ActiveSearchField`.